Dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+# Dockerfile
+# preferred node version chosen here (LTS = 20.9.0 as of 10/24/2023)
+FROM node:20.9.0-alpine
```

Comment:
When we do builds in the AWS environment for this app, it's faster to use the AWS public ECR to fetch base images. Its also guaranteed (or vetted I guess) to work in their ECS container infrastructure

Can you switch this to: `FROM public.ecr.aws/docker/library/node:lts-alpine3.19`

Or if you prefer a different version you can use the select box at the top of this page to find one you want: https://gallery.ecr.aws/docker/library/node

Reply:
That image is accessible from your local machine, so you can use it for both local and AWS builds

Reply:
@briri how can I tell what version of node that image contains?

Reply:
oh nevermind. I think it's still the latest Long Term Support release version